packages/navigation/src/NavigationMenuSelect.tsx

```diff
@@ -28,24 +49,16 @@ const NavigationMenuSelect = <T extends object>(props: CollectionBase<T>) => {
   );
 
   return (
```

Comment:
This diff is a bit complicated:

* Add `nav` with `aria-label` around the `Select`
* Remove the `selected` prop and associated logic (lifted up to the new `useState` above, and set as `value` on the `Select` instead)